packages/math-input/src/fake-react-native-web/view.js

```diff
@@ -1,37 +1,39 @@
-const React = require("react");
-const PropTypes = require("prop-types");
-const {StyleSheet, css} = require("aphrodite");
+// @flow
 
-class View extends React.Component {
-    static propTypes = {
-        ariaLabel: PropTypes.string,
-        children: PropTypes.oneOfType([
-            PropTypes.arrayOf(PropTypes.node),
-            PropTypes.node,
-        ]),
-        // The `dynamicStyle` prop is provided for animating dynamic
-        // properties, as creating Aphrodite StyleSheets in animation loops is
-        // expensive. `dynamicStyle` should be a raw style object, rather than
-        // a StyleSheet.
-        dynamicStyle: PropTypes.any,
-        // The `extraClassName` prop should almost never be used. It gives the
-        // client a way to provide an additional CSS class name, to augment
-        // the class name generated by Aphrodite. (Right now, it's only used to
-        // disable some externally-applied CSS that would otherwise be far too
-        // difficult to override with inline styles.)
-        extraClassName: PropTypes.string,
-        numberOfLines: PropTypes.number,
-        onClick: PropTypes.func,
-        onTouchCancel: PropTypes.func,
-        onTouchEnd: PropTypes.func,
-        onTouchMove: PropTypes.func,
-        onTouchStart: PropTypes.func,
-        role: PropTypes.string,
-        style: PropTypes.any,
-    };
+import {StyleSheet, css} from "aphrodite";
+import * as React from "react";
 
+import type {CSSProperties} from "aphrodite";
+
+type Props = {|
+    ariaLabel?: string,
+    children: React.Node,
+    // The `dynamicStyle` prop is provided for animating dynamic
+    // properties, as creating Aphrodite StyleSheets in animation loops is
+    // expensive. `dynamicStyle` should be a raw style object, rather than
+    // a StyleSheet.
+    dynamicStyle?: CSSProperties,
+    // The `extraClassName` prop should almost never be used. It gives the
+    // client a way to provide an additional CSS class name, to augment
+    // the class name generated by Aphrodite. (Right now, it's only used to
+    // disable some externally-applied CSS that would otherwise be far too
+    // difficult to override with inline styles.)
+    extraClassName?: string,
+    numberOfLines?: number,
+    onClick?: () => void,
+    onTouchCancel?: () => void,
+    onTouchEnd?: () => void,
+    onTouchMove?: () => void,
+    onTouchStart?: () => void,
+    role?: string,
+    style?: CSSProperties,
+|};
+
+class View extends React.Component<Props> {
```

Comment:
This file could likely be replaced with `View` from `@khanacademy/wonder-blocks` core.  No changes req'd in the PR, but something to keep in mind for the wonder-blocks-ifying work y'all be tackling in the future.

Reply:
I 100% agree.  I've made a note and will create a ticket for this.